lib/clouds/vcd_cloud_ops.py

```diff
@@ -17,73 +17,837 @@
 
     @author: Daniel R. Bowers
 '''
+from time import time, sleep
+
+from socket import gethostbyname
 
 from lib.auxiliary.code_instrumentation import trace, cbdebug, cberr, cbwarn, cbinfo, cbcrit
-from libcloud_common import LibcloudCmds
+from lib.auxiliary.data_ops import str2dic, DataOpsException
+from lib.remote.network_functions import Nethashget
+
+from shared_functions import CldOpsException, CommonCloudFunctions 
+
+from libcloud.compute.types import Provider
+from libcloud.compute.providers import get_driver
+from libcloud.compute.types import NodeState
 
-class VcdCmds(LibcloudCmds) :
+import libcloud.security
+
+class VcdCmds(CommonCloudFunctions) :
+    '''
+    TBD
+    '''
     @trace
     def __init__ (self, pid, osci, expid = None) :
-        LibcloudCmds.__init__(self, pid, osci, expid = expid, \
```

Comment:
My apologies. I must have done something wrong with the VCloud adapter. Will just restore it at once!